::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2019 Mirantis Inc., Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Sat, 5 Aug 2017 02:03:01 +0200
Subject: [PATCH] maas: region: skip credentials update

Password update for maas psql database breaks ulterior acesses
to maas-region syncdb.
For now, limit regiond.conf changes to maas_url, and skip
updating credentials.

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 maas/region.sls | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/maas/region.sls b/maas/region.sls
index 066490c..de5054a 100644
--- a/maas/region.sls
+++ b/maas/region.sls
@@ -6,10 +6,9 @@ maas_region_packages:
     - names: {{ region.pkgs }}

 /etc/maas/regiond.conf:
-  file.managed:
-  - source: salt://maas/files/regiond.conf
-  - template: jinja
-  - group: maas
+  file.replace:
+  - pattern: ^maas_url.*$
+  - repl: "maas_url: http://{{ region.bind.host }}:5240/MAAS"
   - require:
     - pkg: maas_region_packages

